springql-core/tests/feat_performance_metrics_report.rs

```diff
@@ -48,47 +47,30 @@ fn _gen_source_input(n: u64) -> impl Iterator<Item = serde_json::Value> {
     })
 }
 
-fn _config(mock: &WebConsoleMock) -> SpringConfig {
+#[cfg(feature = "stub_web_console")]
+fn _config() -> SpringConfig {
     let mut config = SpringConfig::default();
     config.web_console.enable_report_post = true;
     config.web_console.report_interval_msec = 100;
-    config.web_console.host = mock.sock_addr().ip().to_string();
-    config.web_console.port = mock.sock_addr().port();
+    config.web_console.host = "localhost".to_string();
+    config.web_console.port = 12345;
     config
 }
 
-fn _set_callback(
-    mock_builder: WebConsoleMockBuilder,
-    pile: Arc<Mutex<Vec<PostTaskGraphBody>>>,
-) -> WebConsoleMockBuilder {
-    mock_builder.add_callback_post_pipeline(move |body| {
-        pile.lock().unwrap().push(body);
-    })
-}
-
+#[cfg(feature = "stub_web_console")]
 fn start_pile_reports(
     ddls: &[String],
     source_input: ForeignSourceInput,
     test_source: ForeignSource,
     test_sink: &ForeignSink,
-) -> Arc<Mutex<Vec<PostTaskGraphBody>>> {
-    let body_pile = Arc::new(Mutex::new(Vec::new()));
-
-    let mock_builder = WebConsoleMockBuilder::default();
-    let mock_builder = _set_callback(mock_builder, body_pile.clone());
-    let mock = mock_builder.build();
-
-    let config = _config(&mock);
-
-    mock.start();
-
+) {
+    let config = _config();
     let _pipeline = apply_ddls(ddls, config);
     test_source.start(source_input);
     let _sink_received = drain_from_sink(test_sink);
-
-    body_pile
 }
 
+#[cfg(feature = "stub_web_console")]
```

Comment:
this test runs on CI

https://github.com/SpringQL/SpringQL/runs/6945211571?check_suite_focus=true#step:13:428

because our makefile runs default `--all-features`
https://github.com/SpringQL/SpringQL/blob/main/Makefile.toml#L48